src/react/components/pages/homepage/homePage.tsx

```diff
@@ -150,4 +189,16 @@ export default class HomePage extends React.Component<IHomepageProps> {
             throw new AppError(ErrorCode.ProjectDeleteError, "Error deleting project file");
         }
     }
+
+    private convertProject = async (project: IFileInfo) => {
```

Comment:
I recommend renaming many of this variables:
`projects` on the input params is not a project, but rather the v1Project file info.
`projectJson` looks to be the converted project in v2 format rather than a JSON string.